backend/src/StarResults.cjs

```diff
@@ -648,15 +665,18 @@ function flatten(cvr, sections) {
 }
 
 function flattenSingle(cvr) {
+  
   const data = cvr.singleResults;
   const sections = [
     {
       title: data.winners.length > 1 ? "Winner (TIE)" : "Winner",
-      candidates: data.winners
+      candidates: data.winners,
+      votes: data.winnersVotes 
     },
     {
       title: data.losers.length > 1 ? "Runner-Up (TIE)" : "Runner Up",
-      candidates: data.losers
+      candidates: data.losers,
+      votes: data.losersVotes
```

Comment:
I'm a fan of using trailing commas. If ``candidates: data.losers`` already had a comma, then this would only be a single line change, likewise if we give ``votes: data.losersVotes`` a comma then this will make the code diff cleaner the next time we update this

That said it's totally personal preference, I'm fine either way as long as we're consistent in whichever one we choose